bottlecap

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+
+# bottlecap, 'cause it's kinda like cork... get it?
+
+usage() {
+	echo "usage:"
+	echo "bottlecap [--dev] [--runtime $runtime] [--build-dir $build_dir]  [--container $container] assemblerargs..."
+	echo "  --dev: rebuild and reinstall mantle, the build scripts, and the ostree-releng-scipts before running coreos-assembler"
+	echo "  --runtime: (auto) specify whether to use podman or docker. Defaults to podman if it is in \$PATH"
+	echo "  --build-dir: Where to put build artifacts, defaults to the current directory"
+	echo "  --container: which coreos-assembler container to use"
+	echo "  --help: print this help message"
+	echo "Command to be executed:"
+	runtime="echo $runtime"
```

Comment:
I feel like the above two lines should be deleted.

Reply:
I could go either way. It's certainly been useful for debugging `bottlecap` and I think it does a good job of illustrating that `bottlecap` is just a thin wrapper. What about a `---dry-run` option instead?

Reply:
right now it doesn't actually output the command to be executed I don't think. It outputs `Command to be executed:` and then sets the `runtime` equal to `"echo $runtime"`, which doesn't print anything to the screen.

Reply:
I see. Thanks for explaining. I did not understand what you were doing. We discussed that you could also achieve this with `--dry-run`